Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```
@@ -36,127 +36,217 @@
   - type: Sprite
     layers:
     - state: default
-    - state: idpassenger
+    - sprite: Interface/Misc/job_icons.rsi
+      state: Passenger
+      offset: -0.09375, 0.0625 #3 pixels left, 2 pixels up
```

This is an unusual number that's the same for every id, something akin to a constant. I think it's a good opportunity to use an alias.
```suggestion
      offset: &icon-offset -0.09375, 0.0625 #3 pixels left, 2 pixels up
```
Then everywhere else simply use `offset: *icon-offset`.